applications/sasquatch/charts/strimzi-kafka/templates/users.yaml

```diff
@@ -300,3 +300,41 @@ spec:
         host: "*"
         operation: All
 {{- end }}
+{{- if .Values.users.appmetrics.enabled }}
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaUser
+metadata:
+  name: appmetrics
+  labels:
+    strimzi.io/cluster: {{ .Values.cluster.name }}
+spec:
+  authentication:
+    type: scram-sha-512
+    password:
+      valueFrom:
+        secretKeyRef:
+          name: sasquatch
+          key: appmetrics-password
+  authorization:
+    type: simple
+    acls:
+      - resource:
+          type: group
+          name: "*"
+          patternType: literal
+        operation: All
```

Comment:
I copied this from the other users, but I'm not sure why I need it :| Do I actually need it?

Reply:
Yes, in general a user needs privilege to access group resources in the brokers, such as Consumer groups, or even Kafka Connect work groups.